components/SingleArticleContainer.jsx

```diff
@@ -55,11 +81,53 @@ export const SingleArticleContainer = ({ selectedArticle }) => {
         <button onClick={handleIncrease}>+</button>
         <button onClick={handleDecrease}>-</button>
       </p>
-      {hasError && (
+      {voteError && (
         <p className="error">
-          Sorry, your vote couldn't be added. Please try again later
+          Sorry, your vote couldn't be added. Please try again later.
         </p>
       )}
+      {!formOpen ? (
+        <div className="content">
+          <button onClick={handleClick}>New comment</button>
+          {commentAdded ? <p>Comment Posted</p> : null}
+        </div>
+      ) : (
+        <div className="content">
+          <form className="form" onSubmit={handleSubmit}>
+            <div className="row">
+              <label>Username: </label>
+              <input type="text" required={true} />
+            </div>
+            <div className="row">
+              <label>Comment: </label>
+              <input type="text" required={true} />
```

Comment:
I would recommend a text area over an input here. Text areas allow many lines of comments rather than just one.